backend/open_webui/models/models.py

```
@@ -189,98 +199,233 @@ def get_models(self) -> list[ModelUserResponse]:
+    ) -> Optional[ModelModel]:
         try:
-            with get_db() as db:
+            with get_db_context(db) as db:
                 # update only the fields that are present in the model
-                result = (
-                    db.query(Model)
-                    .filter_by(id=id)
-                    .update(model.model_dump(exclude={"id"}))
-                )
+                data = model.model_dump(exclude={"id", "access_grants"})
+                result = db.query(Model).filter_by(id=id).update(data)
```

Ruff (0.15.0)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)